core/modules/layout/plugins/access/entity_id_layout_access.inc

```diff
@@ -26,14 +26,37 @@ class EntityIDLayoutAccess extends LayoutAccessNegatable {
     parent::form($form, $form_state);
 
     $form['entity_id'] = array(
-      '#title' => t('@entity @entity_id_key is', array('@entity' => $this->entity_info['label'], '@entity_id_key' => strtoupper($this->entity_info['entity keys']['id']))),
+      '#title' => t('@entity @entity_id_key is', array(
+        '@entity' => $this->entity_info['label'],
+        '@entity_id_key' => strtoupper($this->entity_info['entity keys']['id']),
+      )),
       '#type' => 'number',
+      '#required' => TRUE,
       '#min' => 1,
       '#default_value' => is_null($this->settings['entity_id']) ? NULL : (int) $this->settings['entity_id'],
       '#weight' => 100,
     );
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  function formValidate($form, &$form_state) {
+    parent::formValidate($form, $form_state);
+
+    $entity_id = $form_state['values']['entity_id'];
+    if (is_numeric($entity_id) && round($entity_id) == $entity_id && $entity_id >= 1) {
```

Comment:
Why this rounding?

Reply:
To make sure it's an integer.

Reply:
Wouldn't typecasting to `(int)` work?

Reply:
...or `intval()` perhaps?

Reply:
`intval()` would work, too; but it's one character longer. Is there a reason why it would be better?

Reply:
Either approach seems fine to me. :shrug: